File: CITS_solution/CITS_App/src/app/models/interview.ts
export interface Interview {
    interviewId: number;
    candidateId: number;
    scheduledDateTime?: Date | null;
    interviewMode: string;
    interviewerId: number;
    stageId: number;
    evaluationType: string;
    status: string;
  meetingLink?: string;


}
//"interviewId": 2,
//  "candidateId": 2,
//    "scheduledDateTime": "2025-08-16T11:00:00",
//      "interviewMode": "Offline",
//        "interviewerId": 2,
//          "stageId": 1,
//            "status": "Completed"
//  },

File: CITS_solution/CITS_WebServices/Models/Interview.cs
using System;
using System.Collections.Generic;

namespace CITS_WebServices.Models;

public partial class Interview
{
    public int InterviewId { get; set; }

    public int CandidateId { get; set; }

    public DateTime? ScheduledDateTime { get; set; }

    public string InterviewMode { get; set; } = null!;

    public int InterviewerId { get; set; }

    public int StageId { get; set; }

    public string Status { get; set; } = null!;
    public string meetingLink { get; set; } = null!;
}
